numpy/_core/code_generators/ufunc_docstrings.py

```diff
@@ -2987,6 +2987,122 @@ def add_newdoc(place, name, doc):
     .. versionadded:: 2.0.0
     """)
 
+add_newdoc('numpy._core.umath', 'matvec',
+    """
+    Matrix-vector dot product of two arrays.
+
+    Let :math:`\\mathbf{A}` be a maxtrix in ``x1`` and :math:`\\mathbf{v}` be
+    a vector in ``x2``. The matrix-vector product is defined as:
```

Comment:
It's good, but if we have an idea, I don't think "a matrix in ``x1``" is very clear about referring to stacks of matrices/vectors.

In fact it would be nice to:
* Add `matvec` and `vecmat` to `matmul` "See Also"
* Probably also to its notes on stacks (IIRC we have those).

Reply:
OK, I tried to reword (will push up in a bit).